hw/top_earlgrey/data/ip/chip_otbn_testplan.hjson

```diff
@@ -0,0 +1,147 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: otbn
+  testpoints: [
+    {
+      name: chip_sw_otbn_isatest
+      descr: '''Run every OTBN instruction and check final state.
+
+                Run the existing OTBN smoke test which uses every instruction
+                (hw/ip/otbn/dv/smoke/smoke_test.s). Check the final register state matches the
+                expected values (in hw/ip/otbn/dv/smoke/smoke_expected.txt). Check OTBN completed
+                successfully without errors.
+
+                Following the smoke test run a new program just to dump out the register state to
+                dmem, check the values do not match the previous values from the smoke test run to
+                ensure the internal state secure wipe has worked.
+             '''
+      stage: V2
+      si_stage: SV1
+      tests: [""]
+      lc_states: ["TEST_UNLOCKED", "DEV", "RMA", "PROD", "PROD_END"]
+      features: [
+        "OTBN.ISA",
+        "OTBN.SECUREWIPE
+      ]
+    }
+    {
+      name: chip_sw_otbn_op
+      desc: '''Verify an OTBN operation.
+
+            - SW test directs the OTBN engine to perform an ECDSA operation.
+            - SW validates the reception of the otbn done interrupt once the operation is complete.
+            - SW verifies the correctness of the result with the expected value which is
+              pre-computed using a reference model.
+            '''
+      stage: V2
+      si_stage: SV2
+      tests: ["chip_sw_otbn_ecdsa_op_irq",
+              "chip_sw_otbn_ecdsa_op_irq_jitter_en"]
+      lc_states: ["TEST_UNLOCKED", "DEV", "RMA", "PROD", "PROD_END"]
+      features: [
+        "OTBN.ISA"
+      ]
+    }
+    {
+      name: chip_sw_otbn_rnd_entropy
+      desc: '''Verify OTBN can fetch RND numbers from the entropy src.
+
+            - SW initializes the entropy subsystem to generate randomness.
+            - SW loads an OTBN app that executes instructions to read the RND bits.
+            - The OTBN app ensures that the values when read consequtively do not match, and its not
+              all 0s or all 1s, as a basic measure to ensure that the entropy subsystem is returning
+              some data.
+            '''
+      stage: V2
+      si_stage: SV1
+      tests: ["chip_sw_otbn_randomness"]
+      lc_states: ["TEST_UNLOCKED", "DEV", "RMA", "PROD", "PROD_END"]
+      features: [
+        "OTBN.RND"
+      ]
+    }
+    {
+      name: chip_sw_otbn_urnd_entropy
+      desc: '''Verify OTBN can fetch URND numbers from the entropy src.
+
+            - Similar to chip_otbn_rnd_entropy, but verifies the URND bits.
+            '''
+      stage: V2
+      si_stage: SV1
+      tests:  ["chip_sw_otbn_randomness"]
+      lc_states: ["TEST_UNLOCKED", "DEV", "RMA", "PROD", "PROD_END"]
+      features: [
+        "OTBN.RND"
+      ]
```

Comment:
I think that this test doesn't check for `OTBN.RND` because URND numbers are not coming from the EDN.
Perhaps this should be a separate feature, something like:
```
name: "OTBN.URND"
desc: "OTBN has a local PRNG that can provide 256-bit pseudo-random numbers without stalling."
```

Reply:
That makes sense, but I'm a little worried about the proliferation of features! Maybe we should rename the feature to something like "OTBN.RANDOM" and make it cover both?

Reply:
URND is seeded from the EDN though. I think I'll take @rswarbrick's suggestion and rename the feature to OTBN.RANDOM